src/common.ts

```diff
@@ -61,6 +61,9 @@ export enum ContractAddressId {
   helper = "HELPER",
   allowlist = "ALLOW_LIST_REGISTRY",
   partner = "PARTNER_TRACKER",
+  zapperZapIn = "ZAPPER_ZAP_IN",
+  zapperZapOut = "ZAPPER_ZAP_OUT",
+  pickleZapIn = "PICKLE_ZAP_IN",
```

Comment:
Usually string enum in TS are capitalised, https://www.typescriptlang.org/docs/handbook/enums.html#string-enums -- reckon it might be too late for that here :)

Reply:
Yeah, wanted to be consistent with the rest. Maybe a good refactor for other PR?

Reply:
It could be, depending on how these enums are being used. If only in this service all good, if used elsewhere I'm afraid we might be stuck with these ones for the time being :)

No action needed